recipe/meta.yaml

```diff
@@ -37,6 +37,10 @@ test:
     - pytest
   imports:
     - pywt
+  commands:
+    {% set label = "'full'" %}
+    {% set tests = "['pywt']" %}
+    - python -c "import sys; import pywt; sys.exit(not pywt.test(verbose=2, label={{ label }}, tests={{ tests }}, extra_argv=['--durations=50']))"
```

Comment:
Had copied the `extra_argv=['--durations=50']` argument from `scipy`, but IDK if we need it here. Happy to leave it or drop it

Reply:
I'd say let's leave it, it is useful in detecting tests that for some reason are really slow. This can happen due to CI config (virtual drive mounting etc.) or emulation. We can then address that in the main repo.